src/rules/declaration-property-value-no-unknown/__tests__/index.js

```diff
@@ -323,8 +323,28 @@ testRule({
         }
     }
     `,
-    description: "Nested properties in nested declarations using functions."
-  }
+      description: "Nested properties in nested declarations using functions."
+    },
+    {
+      code: `
+    div {
+      border: solid #000 {
+        width: 1px 1px 1px 7px;
+      }
+    }
+    `,
+      description: "Nested properties and shorthand values."
+    },
+    {
+      code: `
+      @use "qux";
+      .b {
+        margin: $foo;
+        margin: (-$foo);
```

Comment:
Related to my other comment, is `margin: (-$foo);` something that we should support in `isDollarVar`, or would it be better if the rule took care of parsing this value?

The reason why I'm asking is that it's quite easy to end up having to add more cases that include some exotic syntax that is valid SCSS if we choose to start supporting them.

Reply:
Makes sense! Since the function is named isDollarVar, it should return whether the argument is strictly a variable (not if it's containing one).